src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/commands.py

```diff
@@ -7,14 +7,27 @@
     get_sql_servers_operations,
     get_sql_firewall_rules_operations,
     get_sql_databases_operations,
-    get_sql_elastic_pools_operations)
+    get_sql_elastic_pools_operations,
+    get_sql_capabilities_operations)
 from azure.cli.core.sdk.util import (
     create_service_adapter,
     ServiceGroup)
 
 custom_path = 'azure.cli.command_modules.sql.custom#{}'
 
 ###############################################
+#                sql capabilities             #
+###############################################
+
+capabilities_operations = create_service_adapter(
+    'azure.mgmt.sql.operations.capabilities_operations',
+    'CapabilitiesOperations')
+
+with ServiceGroup(__name__, get_sql_capabilities_operations, capabilities_operations, custom_path) as s:
+    with s.group('sql') as c:
+        c.custom_command('show-capabilities', 'capabilities_get')
```

Comment:
You mention that this data is impossible or very difficult to filter with JMESPath.  In that case, how would you use TSV format (which relies exclusively on --query)?  Since this is an informational command, a table_formatter would be highly recommended.  You might also consider a generic transform for this command to that flattens the tree structure to make TSV and Table formatting easier. You're already doing so much custom work here 😃 

I just see that someone might want to dump their subscription info to TSV and open in Excel or something since it doesn't really change.